src/Algebra/Module/Structures.agda

```diff
@@ -210,10 +211,12 @@ module _ (ring : Ring r ℓr)
       ( isGroup    to +ᴹ-isGroup
       ; inverseˡ   to -ᴹ‿inverseˡ
       ; inverseʳ   to -ᴹ‿inverseʳ
-      ; uniqueˡ-⁻¹ to uniqueˡ‿-ᴹ
-      ; uniqueʳ-⁻¹ to uniqueʳ‿-ᴹ
       )
 
+    open AbelianGroupProperties record { isAbelianGroup = +ᴹ-isAbelianGroup } public
+      using () renaming (inverseˡ-unique to uniqueˡ‿-ᴹ; inverseʳ-unique to uniqueʳ‿-ᴹ)
```

Comment:
If `uniqueˡ-⁻¹` didn't belong in `Algebra.Structures`, then morally these shouldn't belong in `Algebra.Module.Structures`? If the `Properties` file already exists then they could be transferred there? Otherwise we could leave a comment?

Reply:
Thanks! I think this represents a wobble/failure of nerve on my part, while *not* trying to solve the problem of violating the meta-level principle '`Structures` shouldn't depend on `Bundles`'.

But in a certain sense that is *already* a bigger problem: that `Algebra.Properties.X` shouldn't be parameterised on a `Bundle` `x : X, but rather on the underlying `Structure` `isX : IsX` which supplies the 'axiomatisation' of `X`s...

Reply:
But on that basis, I'd be happy to put this back in `DRAFT`, or even to close this PR, against a wider rethink of these issues...

Reply:
No, no I think that this is an improvement over what we currently have.

Reply:
OK, I'll see what/how best to tackle the above, and then deal with the outstanding merge conflict.

Reply:
So... revisiting this now, I'd happily move the definitions entirely to `Algebra.Module.Properties.*module`, but for the fact that it would mean leaving the names as they are currently in-place, and duplicating them in `Algebra.Module.Properties` (so introducing a possible bug on `import`), because the `renaming` actually chooses the 'correct' names.

For `Group`, the deprecated names aren't identical to the 'correct' names in `Algebra.Properties.Group`, so the issue doesn't arise, and I can't really face introduce 'fake' names for `Module` simply for the sake of specialising the deprecation in-place in order to point to the 'correct' version elsewhere. 

Or is that the 'right' thing to do?